admin/load.php

```diff
@@ -503,3 +508,77 @@ function perflab_dismiss_wp_pointer_wrapper() {
 	check_ajax_referer( 'dismiss_pointer' );
 }
 add_action( 'wp_ajax_dismiss-wp-pointer', 'perflab_dismiss_wp_pointer_wrapper', 0 );
+
+/**
+ * Callback function to handle admin scripts.
+ *
+ * @since n.e.x.t
+ */
+function perflab_enqueue_modules_page_scripts() {
+	wp_enqueue_script( 'updates' );
+
+	wp_localize_script(
+		'updates',
+		'_wpUpdatesItemCounts',
+		array(
+			'settings' => array(
+				'totals' => wp_get_update_data(),
+			),
+		)
+	);
+
+	wp_enqueue_script( 'thickbox' );
+	wp_enqueue_style( 'thickbox' );
+}
+
+/**
+ * Callback function hooked to admin_action_perflab_activate_plugin to handle plugin activation.
+ *
+ * @since n.e.x.t
+ */
+function perflab_activate_plugin() {
+	// Do not proceed if plugin query arg is not present.
+	if ( empty( $_GET['plugin'] ) ) {
+		return;
+	}
+
+	// The plugin being activated.
+	$plugin = sanitize_text_field( $_GET['plugin'] );
+
+	if ( check_admin_referer( "perflab_activate_plugin_{$plugin}" ) && current_user_can( 'activate_plugin', $plugin ) ) {
+		// Activate the plugin in question and return to prior screen.
+		activate_plugins( $plugin );
+		$referer = wp_get_referer();
+		if ( wp_safe_redirect( $referer ) ) {
+			exit;
+		}
+	}
```

Comment:
The UX would be a bit better by failing with an error rather than silently. Note that `check_admin_referer()` does that automatically (via `wp_nonce_ays()`).
```suggestion
	check_admin_referer( "perflab_activate_plugin_{$plugin}" );

	if ( ! current_user_can( 'activate_plugin', $plugin ) ) {
		wp_die( __( 'Sorry, you are not allowed to activate this plugin.', 'default' ) );
	}

	// Activate the plugin in question and return to prior screen.
	activate_plugins( $plugin );
	$referer = wp_get_referer();
	if ( wp_safe_redirect( $referer ) ) {
		exit;
	}
```